artacle-swagger.yaml

```diff
@@ -825,6 +827,641 @@ paths:
+                          example: 0.0414
+                        mintPriceUSD:
+                          type: number
+                          description: Token mint price in USD
+                          format: float
+                          example: 102.768876
+                        listingPrice:
+                          type: number
+                          description: Token current listing price
+                          format: float
+                          example: 0.0414
+                        listingPriceUSD:
+                          type: number
+                          description: Token current listing price in USD
+                          format: float
+                          example: 102.768876
+                        listingTimestamp:
+                          type: number
+                          format: uint64
+                          description: Unix timestamp when token is listed
+                          example: 1705667581
+                        isListed:
+                          type: number
+                          enum:
+                            - 0
+                            - 1
+                          description: Indicates if token is currently listed
+                          example: 1
+                        lastSalePrice:
+                          type: number
+                          description: Token last sale price
+                          format: float
+                          example: 0.0414
+                        lastSalePriceUSD:
+                          type: number
+                          description: Token last sale price in USD
+                          format: float
+                          example: 102.768876
+        '400':
+          $ref: "#/components/responses/BadRequestError"
+        '401':
+          $ref: "#/components/responses/UnauthorizedError"
+        '403':
+          $ref: "#/components/responses/ForbiddenError"
+        '404':
+          $ref: "#/components/responses/NotFoundError"
+        '429':
+          $ref: "#/components/responses/TooManyRequestsError"
+        '500':
+          $ref: "#/components/responses/InternalServerError"
+
+  /dashboard/v1:
+    get:
+      tags:
+        - Dashboard
+      summary: Get collections dashboard information
+      description: Returns collections dashboard information
+      parameters:
+        - name: apiKey
+          in: query
+          required: true
+          description: API access key
+          schema:
+            type: string
+      responses:
+        '200':
+          description: Success
+          content:
+            application/json:
+              schema:
+                type: object
+                required:
+                  - data
+                properties:
+                  data:
+                    type: array
+                    description: Array of collections data
+                    items:
+                      type: object
```

Comment:
I'm rather not sure if we should list all fields here. Maybe we can omit listing them? Cause they will be extended in the future, and they are pretty much self-descriptive.